### PR TITLE
libselinux: LABEL_BACKEND_ANDROID add option to enable it

### PR DESCRIPTION
**Pull request by Thaodan**
Add option to just enable the android label backend without disabling anything else.

### CODE REVIEW

**Comment by bachradsusi** [comment]
Thanks for the patch. Please send it for review to [SELinux mailing](https://lore.kernel.org/selinux) list at selinux@vger.kernel.org see https://github.com/SELinuxProject/selinux/blob/master/CONTRIBUTING.md

If you can't for some reason, I can do it for you.

**Comment by God-knows-all** [comment]
Do it please

**Review by God-knows-all — APPROVED** [review]

**Comment by Thaodan** [comment]
Rebased, use better wording.

File: libselinux/Makefile [hunk 1]
```diff
@@ -4,6 +4,7 @@ PKG_CONFIG ?= pkg-config
 DISABLE_SETRANS ?= n
 DISABLE_RPM ?= n
 ANDROID_HOST ?= n
+LABEL_BACKEND_ANDROID ?= n
```
Comment (God-knows-all) [inline libselinux/Makefile:7]: #268

**Review by God-knows-all — COMMENTED** [review]
> `` #268 #265

**Pull request merged**